ebook/2_html2epub.sh

```diff
@@ -0,0 +1,41 @@
+# by Torben Menke https://entorb.net
+
+# run from within ebook dir via 
+# bash 2_html2epub.sh
+
```

Comment:
Add a hash-bang line, please.

Reply:
good point, added to the .py as well

Reply:
Thanks; I didn't see a reply to my query, does it have to be bash, or can it just be `/bin/sh`?

Reply:
I assume any shell should work. The only "shell logic" I am using is if ... than.
For the .sh script I added this hash-bang:
#!/usr/bin/env bash

Reply:
Since it's `sh`-compatible, the hash-bang line should be just `#!/bin/sh` (this path is actually mandated by POSIX, no need for `env`).

Reply:
ah, again I learned something new, thanks.
changed